### DISCUSSION TITLE
Issue with Updating User Status in Supabase auth.users table

### DISCUSSION THREAD

**Discussion by UmaDOINFOTECH** [supabase/supabase#27840] [Questions]
I'm encountering an issue while trying to update the user status using the Supabase Flutter SDK in my Flutter application.  I have altered the auth.users table to include a status column. I have a Flutter application where I need to update the status of a user in both the public.users and auth.users tables stored in Supabase. 

I need assistance understanding how to correctly update the user status in the auth.users table using the Supabase Flutter SDK. Specifically, how can I pass the userId parameter correctly to update the user status based on the user's UUID?

Any guidance or suggestions on resolving this issue would be greatly appreciated! Thank you.

**Comment by GaryAustin1** [comment]
You cannot change the auth.users table at all.  You can use app or user metadata columns for your own stuff with additional json info.  Otherwise you need a public profile table like the user management guides show for other details.

If I recall, the gotrue (auth) server will actually check the row to make sure there are not additional or missing columns and they are the correct types.
